.github/renovate.json5

```
@@ -519,6 +519,7 @@
       "fileMatch": [
         "^test/kubernetes-test\\.sh$",
         "^test/packet/scripts/install\\.sh$"
+        "^.github/actions/set-env-variables/action.yml$"
```

There's a syntax error in the list. Also, let's match on the dot in the file name.

```suggestion
        "^test/packet/scripts/install\\.sh$",
        "^.github/actions/set-env-variables/action\\.yml$"
```

---

whops, fixed :+1: